unity/Scripts.csproj

```diff
@@ -644,23 +659,20 @@
     <Reference Include="System.Xml.Serialization">
       <HintPath>C:/Program Files/Unity/Hub/Editor/2019.4.22f1/Editor/Data/NetStandard/compat/2.0.0/shims/netfx/System.Xml.Serialization.dll</HintPath>
     </Reference>
+    <Reference Include="Unity.InputSystem">
+      <HintPath>E:/Exchange/ZHAW/FS2021/PSIT4/Hexxle/hexxle_game/unity/Library/ScriptAssemblies/Unity.InputSystem.dll</HintPath>
+    </Reference>
+    <Reference Include="Unity.TextMeshPro">
+      <HintPath>E:/Exchange/ZHAW/FS2021/PSIT4/Hexxle/hexxle_game/unity/Library/ScriptAssemblies/Unity.TextMeshPro.dll</HintPath>
+    </Reference>
     <Reference Include="UnityEditor.UI">
-      <HintPath>C:/Users/ricar/Documents/GitHub/hexxle_game/unity/Library/ScriptAssemblies/UnityEditor.UI.dll</HintPath>
+      <HintPath>E:/Exchange/ZHAW/FS2021/PSIT4/Hexxle/hexxle_game/unity/Library/ScriptAssemblies/UnityEditor.UI.dll</HintPath>
```

Comment:
This seems to be a weird line that changes all the time, might be good to try and see if we can ignore this file

Reply:
That's true, however this file also includes included libraries and other somewhat important information, I'm not sure if ignoring the whole file is a good idea.